src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
![image](https://github.com/mate-academy/js_stateful-object/assets/139622699/c01e943d-678a-4bae-b8b5-4de4fbab7126)

Reply:
Not fixed.